### PR TITLE
Cdc 804: pkg version number

### PR DESCRIPTION
**Pull request by javila-mechanicode**
## Additions

- 

## Removals

-

## Updates

- pkg naming convention - gov.cdc.dataexchange.pkgname
- pkg numbering convention - starting on 1.0.0-SNAPSHOT like the function apps
- following README of azure-maven-plugins for plugin config

## Testing Steps

-

## Notes

-

### CODE REVIEW

File: svc-cloud-transport/pom.xml [hunk 1]
```diff
@@ -289,15 +290,16 @@
                     <appServicePlanName>${functionServicePlan}</appServicePlanName>
                     <subscriptionId>${subscriptionID}</subscriptionId>
                     <region>${functionRegion}</region>
-
+
```
Comment (emmanuel-apau) [inline svc-cloud-transport/pom.xml:293]: need to add the <auth> tag like the other functions https://github.com/microsoft/azure-maven-plugins/wiki/Authentication#service-principles-in-settings.xml

File: svc-cloud-transport/pom.xml [hunk 2]
```diff
@@ -279,18 +279,21 @@
             <plugin>
                 <groupId>com.microsoft.azure</groupId>
                 <artifactId>azure-webapp-maven-plugin</artifactId>
-                <version>2.2.0</version>
+                <version>2.9.0</version>
```
Comment (emmanuel-apau) [inline svc-cloud-transport/pom.xml:282]: latest version = [2.11.0](https://mvnrepository.com/artifact/com.microsoft.azure/azure-webapp-maven-plugin/2.11.0)

**Comment by javila-mechanicode** [comment]
@mscaldas2012  - ready for review

**Review by emmanuel-apau — APPROVED** [review]
lgtm

**Review by marcia-schulman — APPROVED** [review]

**Pull request merged**